### DISCUSSION TITLE
`test-bot --only-setup` failing in GH Action

### DISCUSSION THREAD

**Discussion by hughrun** [Homebrew/discussions#1435] [Tap maintenance and Homebrew development]
I've recently created a tap for a small Rust project, and tried to follow the [Homebrew tap with bottles uploaded to GitHub Releases](https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/) guide.

The `test-bot (Ubuntu latest)` action is consistently failing when a PR is created, at the `test-bot --only-setup` command:
```
Run brew test-bot --only-setup
  brew test-bot --only-setup
  shell: /usr/bin/bash -e {0}
Error: No such file or directory
```

My formula is:
```ruby
class LetteRs < Formula
  desc "A CLI tool to make static site publishing less painful." 
  homepage "https://github.com/hughrun/lette.rs"
  url "https://github.com/hughrun/lette.rs/archive/refs/tags/v1.2.2.tar.gz"
  sha256 "66459e9d5b52c8c4489061b4337a918d31eb70499c4d9193da990f6fc33e8a1e"
  version "1.2.2"
  license "GPL-3.0"

  depends_on "rust" => :build

  def install
    system "cargo", "install", *std_cargo_args
  end

end 
```

I assume this is something I'm doing wrong but I cannot work out what it is. Any advice would be gratefully received!

**Comment by carlocab** [comment]
Tried creating a new tap with `brew tap-new` and opened a PR with your formula: https://github.com/carlocab/homebrew-tap-new-test/pull/1

I can't reproduce the problem you're having, so I can't say anything about fixing it. (CI does fail in my PR, but it doesn't fail in the `--only-setup` step.)

**Reply by carlocab** [reply]
~~Hmm, maybe the `.` in the formula name is causing your issue. I had that as a `-` in my initial test. Let me try changing it.~~ Nope, also works.
